druid-shell/src/platform/gtk/window.rs

```diff
@@ -90,6 +90,8 @@ macro_rules! clone {
 #[derive(Clone, Default)]
 pub struct WindowHandle {
     pub(crate) state: Weak<WindowState>,
+    // Ensure that we don't implement Send, because it isn't actually safe to send the WindowState.
+    marker: std::marker::PhantomData<*const ()>,
```

Comment:
I think we should enforce this for all platforms, to ensure the behaviour is consistent, so maybe put it in the common `WindowHandle`, not the platform ones?

Reply:
Yes, that's the purpose of the static assert on the common WindowHandle: to enforce the cross-platform behavior. It just so happens that only the GTK one needed an extra marker.